alfresco-docker-transform-misc/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Image provides a container in which to run miscellaneous transformations for Alfresco Enterprise Content Services.
+
+# The container is only intended to be used with the Alfresco Enterprise editon which is covered by https://www.alfresco.com/legal/agreements and https://www.alfresco.com/terms-use.
```

Comment:
This line was added for Tika. It should not be here but there may be other licenses for 3rd part jars the the 'misc' T-Engine uses.